code/modules/robotics/robot/module_tool_creator/modules.dm

```diff
@@ -86,110 +85,131 @@
 		/obj/item/reagent_containers/food/drinks/drinkingglass/icing,
 		/obj/item/fishing_rod/cybernetic,
 		/obj/item/storage/fish_box/small,
+		/obj/item/device/camera_viewer/public,
+		/obj/item/pen/omni,
+		/obj/item/paper_bin/robot,
 	)
 
 /datum/robot/module_tool_creator/recursive/module/common
 	definitions = list(
+		/obj/item/portable_typewriter/borg,
+		/obj/item/robojumper,
+		/obj/item/device/analyzer/atmospheric/upgraded,
+		/obj/item/device/reagentscanner,
 		/obj/item/device/light/flashlight,
-		/obj/item/tool/omnitool/silicon,
 		/obj/item/device/analyzer/healthanalyzer/upgraded,
-		/obj/item/device/reagentscanner,
-		/obj/item/device/analyzer/atmospheric/upgraded,
-		/obj/item/robojumper,
-		/obj/item/portable_typewriter/borg,
-	)
-
-/datum/robot/module_tool_creator/recursive/module/construction_ai
-	definitions = list(
-		/obj/item/rcd,
-		/obj/item/electronics/scanner,
-		/obj/item/electronics/soldering,
-		/obj/item/room_planner,
-		/obj/item/room_marker,
-		/datum/robot/module_tool_creator/item_type/amount/cable_coil,
-	)
-
-/datum/robot/module_tool_creator/recursive/module/construction_worker
-	definitions = list(
-		/obj/item/weldingtool,
-		/obj/item/electronics/scanner,
-		/obj/item/electronics/soldering,
-		/obj/item/room_planner,
-		/obj/item/room_marker,
-		/obj/item/extinguisher/large/cyborg,
-		/obj/item/rcd,
-		/obj/item/deconstructor/borg,
-		/datum/robot/module_tool_creator/item_type/amount/steel_tile,
-		/datum/robot/module_tool_creator/item_type/amount/steel_rod,
-		/datum/robot/module_tool_creator/item_type/amount/steel_sheet,
-		/datum/robot/module_tool_creator/item_type/amount/glass_sheet,
-		/datum/robot/module_tool_creator/item_type/amount/cable_coil,
+		/obj/item/tool/omnitool/silicon,
 	)
 
 // engineer. mechanic.
+// DOES NOT inherit common tools
 /datum/robot/module_tool_creator/recursive/module/engineering
 	definitions = list(
-		/obj/item/atmosporter,
+		/obj/item/portable_typewriter/borg,
+		/obj/item/device/analyzer/healthanalyzer/upgraded,
+		/obj/item/device/analyzer/atmospheric/upgraded,
+		/obj/item/device/reagentscanner,
```

Comment:
I recommend moving the reagent scanner above the atmospheric analyzer, due to the analyzer being used more for making sure an area is repressurized and whatnot.

Reply:
Moved as suggested.